React/Views/RCTModalHostView.m

```diff
@@ -176,7 +168,7 @@ - (BOOL)hasAnimationType
 
 - (void)setTransparent:(BOOL)transparent
 {
-  if ([self isTransparent] != transparent) {
+  if (self.transparent != transparent) {
```

Comment:
Generally ObjC prefers using bracket syntax for synthesized getters, or dot syntax for property reference directly. This is the most common practice, so I am going to leave it as is.